Lib/test/clinic.test.c

```diff
@@ -5464,3 +5465,274 @@ docstr_fallback_to_converter_default(PyObject *module, PyObject *const *args, Py
 static PyObject *
 docstr_fallback_to_converter_default_impl(PyObject *module, str a)
 /*[clinic end generated code: output=ae24a9c6f60ee8a6 input=0cbe6a4d24bc2274]*/
+
+
+/*[clinic input]
+test_deprecate_positional_use_1
+    pos: object
+    * [from 3.14]
+    optarg: object = None
+      has a default value, unlike test_deprecate_positional_use_2
+[clinic start generated code]*/
+
+PyDoc_STRVAR(test_deprecate_positional_use_1__doc__,
+"test_deprecate_positional_use_1($module, /, pos, optarg=None)\n"
+"--\n"
+"\n"
+"\n"
+"\n"
+"  optarg\n"
+"    has a default value, unlike test_deprecate_positional_use_2");
+
+#define TEST_DEPRECATE_POSITIONAL_USE_1_METHODDEF    \
+    {"test_deprecate_positional_use_1", _PyCFunction_CAST(test_deprecate_positional_use_1), METH_FASTCALL|METH_KEYWORDS, test_deprecate_positional_use_1__doc__},
+
+static PyObject *
+test_deprecate_positional_use_1_impl(PyObject *module, PyObject *pos,
+                                     PyObject *optarg);
+
+static PyObject *
+test_deprecate_positional_use_1(PyObject *module, PyObject *const *args, Py_ssize_t nargs, PyObject *kwnames)
+{
+    PyObject *return_value = NULL;
+    #if defined(Py_BUILD_CORE) && !defined(Py_BUILD_CORE_MODULE)
+
+    #define NUM_KEYWORDS 2
+    static struct {
+        PyGC_Head _this_is_not_used;
+        PyObject_VAR_HEAD
+        PyObject *ob_item[NUM_KEYWORDS];
+    } _kwtuple = {
+        .ob_base = PyVarObject_HEAD_INIT(&PyTuple_Type, NUM_KEYWORDS)
+        .ob_item = { &_Py_ID(pos), &_Py_ID(optarg), },
+    };
+    #undef NUM_KEYWORDS
+    #define KWTUPLE (&_kwtuple.ob_base.ob_base)
+
+    #else  // !Py_BUILD_CORE
+    #  define KWTUPLE NULL
+    #endif  // !Py_BUILD_CORE
+
+    static const char * const _keywords[] = {"pos", "optarg", NULL};
+    static _PyArg_Parser _parser = {
+        .keywords = _keywords,
+        .fname = "test_deprecate_positional_use_1",
+        .kwtuple = KWTUPLE,
+    };
+    #undef KWTUPLE
+    PyObject *argsbuf[2];
+    Py_ssize_t noptargs = nargs + (kwnames ? PyTuple_GET_SIZE(kwnames) : 0) - 1;
+    PyObject *pos;
+    PyObject *optarg = Py_None;
+
+    #if PY_VERSION_HEX >= 0x030e00C0
+    #  error "In /Users/erlend.aasland/src/cpython.git/Lib/test/clinic.test.c, update parameter(s) optarg in the clinic input of 'test_deprecate_positional_use_1' to be keyword-only."
```

Comment:
Hardcoding your own name into the test suite? 🤨

Reply:
Oh my 😅

Reply:
Fixed (temporarily)